tests/order.py

```diff
@@ -81,4 +80,54 @@ def test_remove_product_from_order(self):
 
     # TODO: Complete order by adding payment type
 
-    # TODO: New line item is not added to closed order
\ No newline at end of file
+    def test_add_payment_type_to_order(self):
+        """
+        Ensure we can add a payment type to an order.
+        """
+        self.test_add_product_to_order()
+
+        # add payment_type to order
+        url = "/orders/1"
+        data = {"payment_type: 1"}
```

Comment:
Double check the location of the quote marks here and make sure the there's a payment type with an id of one in the test database. Take a look in the `setup` in here for how to seed the test database